src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 300;
+  src: url(/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-size: 16px;
+  font-weight: 900;
+  src: url(/fonts/Avenir-Heavy.ttf);
+}
+
+.input {
+  color: #3d4e61;
+  display: block;
+  font-style: normal;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  outline: none;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  background-size: 19px;
+  background-position: 25px 24px;
+  width: 100%;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+}
+
+.input:hover {
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  color: #3d4e61;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-family: "Avenir", sans-serif;
+  font-weight: 900;
+  border-radius: 3px;
+  text-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input--small {
```

Comment:
Please remove repeated style rules which you already use in .input class
![image](https://user-images.githubusercontent.com/54936044/228382731-d7b4b043-b903-48bb-bb61-d6e7f137b724.png)